docs/ref/settings.txt

```diff
@@ -1873,10 +1873,9 @@ for :doc:`managing stored files </topics/files>`. It must end in a slash if set
 to a non-empty value. You will need to :ref:`configure these files to be served
 <serving-uploaded-files-in-development>` in both development and production.
 
-In order to use ``{{ MEDIA_URL }}`` in your templates, you must have
+If you want to directly use ``{{ MEDIA_URL }}`` in your templates, add
```

Comment:
I think the current wording "use" is okay (considering it shows it as a template variable), but if you think that's better, sure.